packages/react-divider/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@fluentui/react-divider",
+  "version": "9.0.0-alpha.2",
+  "description": "Fluent UI component to visually separate content.",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.2",
```

Comment:
Looks like you have at least one dep version out of sync. Once you've merged with master, try running `yarn syncpack:fix`. (This *usually* fixes things and should work fine in this case, but it can sometimes choose the wrong version--so you just have to look over the changes and make sure it looks right.)

Reply:
updated